unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDView.cs

```diff
@@ -122,6 +122,12 @@ private void Awake()
 
         isOpen = false;
         arePanelsInitialized = false;
+        characterPreviewRotation.OnPointerAction += SetCanScroll;
```

Comment:
Please remember to unsubscribe from events. It may seem useless as the view seem to be instanced only once, but this issue can create hard-to-fix flaky tests.

Reply:
Yes sorry I don't know why I keep on forgetting, fixed with the latest commit